python/cuml/common/array.py

```diff
@@ -222,6 +232,16 @@ def to_output(self, output_type='cupy'):
                 else:
                     raise ValueError('cuDF unsupported Array dtype')
 
+    def serialize(self):
```

Comment:
This is just for Dask, right? It doesn't apply to pickling?

Reply:
This is for Dask and is following the same convention used in cuDF. For example, here's where [`Buffer` implements this](  https://github.com/rapidsai/cudf/blob/ad36bdd18ea1d5e1db366b40761a86a4ea526e8c/python/cudf/cudf/core/buffer.py#L106-L112 ), which is what this is using for the first serialization pass.

Right this has nothing to do with pickling the data.